src/bundle/Resources/config/views.yml

```diff
@@ -27,3 +27,5 @@ system:
                     match: true
                     params:
                         viewbaseLayout: '@EzPlatformAdminUi/layout.html.twig'
+        fielddefinition_edit_templates:
```

Comment:
`field_definition_edit_templates` ?

Reply:
Nope, this is proper name defined in the kernel.

Reply:
ok, sorry :)